tools/pythonpkg/tests/fast/test_relation.py

```diff
@@ -153,6 +153,35 @@ def test_df_proj(self,duckdb_cursor):
         rel = duckdb.project(test_df, 'i')
         assert rel.execute().fetchall() == [(1,), (2,), (3,), (4,)]
 
+    def test_project_on_types(self, duckdb_cursor):
+        con = duckdb_cursor
+        con.sql("""
+            create table tbl(
+                c0 BIGINT,
+                c1 TINYINT,
+                c2 VARCHAR,
+                c3 TIMESTAMP,
+                c4 VARCHAR,
+                c5 STRUCT(a VARCHAR, b BIGINT)
+            )
+            """)
+        rel = con.table("tbl")
+        # select only the varchar columns
+        projection = rel.project(["varchar"])
+        assert projection.columns == ["c2", "c4"]
+
+        # select bigint, tinyint and a type that isn't there
+        projection = rel.project([duckdb.bigint, "tinyint", con.struct_type({'a': duckdb.varchar, 'b': duckdb.tinyint})])
```

Comment:
Does it make sense to have some mapping of `Python types -> DuckDB types`, for example:

```python
int -> duckdb.bigint
str -> duckdb.varchar
float -> duckdb.double
bytearray -> blob
```